src/instruments/src/RMP/Components/VhfRadioPanel.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { StandbyFrequency } from './StandbyFrequency';
+import { useSplitSimVar } from '../../Common/simVars';
+import { RadioPanelDisplay } from './RadioPanelDisplay';
+import { useInteractionEvent } from '../../Common/hooks';
+
+/**
+ *
+ */
+interface Props {
+    side: string,
+    transciever: number,
+}
+
+/**
+ *
+ */
+const useActiveVhfFrequency = (transciever: number) => {
+    const variableReadName = `COM ACTIVE FREQUENCY:${transciever}`;
+    const variableWriteName = `K:COM${transciever === 1 ? '' : transciever}_RADIO_SET_HZ`;
+    return useSplitSimVar(variableReadName, 'Hz', variableWriteName, 'Hz', 100);
+}
+
+/**
+ *
+ */
+const useStandbyVhfFrequency = (side: string, transciever: number) => {
+    let variableReadName = `COM STANDBY FREQUENCY:${transciever}`;
+    let variableWriteName = `K:COM${transciever === 1 ? '' : transciever}_STBY_RADIO_SET_HZ`;
+
+    // Use custom SimVars for abnormal standby frequency.
+    // Allows true-to-life independent standby frequencies per RMP.
+    // @todo, if we ever add a third RMP (e.g. C), update this.
```

Comment:
**thought**: I'm not sure if I'd add a `@todo` for this. As IDEs will show these in the "todo list" whilst they're not really something to do now.